### DISCUSSION TITLE
Features and Ideas!

### DISCUSSION THREAD

**Discussion by D3Ext** [D3Ext/WEF#6] [Ideas]
## 👋 Welcome

This Discussion is used to share ideas and more about **WEF**

If you have any feature or idea and you want  me to add it, you can contact me here and I'll try to answer as soon as possible.

**Comment by ultrazar** [comment]
It would be cool to have the krack attack implemented in the framework. Although nowadays it is patched in most devices it may work against some old IT hardware ...

**Reply by D3Ext** [reply]
it's already implemented, when you catch a handshake for example during deauthentication attack, the tool will ask you to crack the handshake. Anyway I'll implement it as another command so you can crack handshakes you already have.

**Comment by renwixx** [comment]
Hello! I modified waf file because it cannot install via termux on my phone. /system sector is read-only, so i installed waf via termux after replacing all "/usr/share" matches to "/data/data/com.termux/files/usr/share". Hope it helps! 
![Screenshot_20240917-150749](https://github.com/user-attachments/assets/cbd6485f-b296-413a-8554-69661545674e)

**Comment by D3Ext** [comment]
Thank you for the idea. I haven't tested WEF on Termux so I don't know if it works perfectly. Anyways, I will modify the script in order to adapt the root path from /usr/share to Termux path

**Reply by renwixx** [reply]
thanks!
